config/config.go

```diff
@@ -190,6 +191,9 @@ type ConnectorDTO struct {
 // GWConnection defines Groundwork Connection configuration
 type GWConnection clients.GWConnection
 
+// GWConnection defines Groundwork Connection configuration
```

Comment:
should be TCGConnection 

https://github.com/golang/go/wiki/CodeReviewComments#doc-comments

Reply:
Fixed.